demo/admin/src/products/future/generated/CombinationFieldsTestProductsGrid.tsx

```diff
@@ -131,6 +132,40 @@ export function ProductsGrid({ toolbarAction, rowAction }: Props): React.ReactEl
             flex: 1,
             minWidth: 150,
         },
+        {
+            field: "currencyAndNumber",
+            headerName: intl.formatMessage({ id: "product.currencyAndNumber", defaultMessage: "Price (currency and number)" }),
+            filterable: false,
+            sortable: false,
+            renderCell: ({ row }) => {
+                return (
+                    <GridCellContent
+                        primaryText={
+                            typeof row.price === "number" ? (
+                                <FormattedNumber
+                                    value={row.price}
+                                    minimumFractionDigits={2}
+                                    maximumFractionDigits={2}
+                                    style="currency"
+                                    currency="EUR"
+                                />
+                            ) : (
+                                "-"
```

Comment:
could this default to simply rendering row.price (whatever it contains)?

Reply:
It would just be an empty string, because `row.price` would be `null`/`undefined`...

Reply:
Do you think we should remove the default empty message `"-"`, since we can set `emptyValue` in the config anyway?

Reply:
Yes, this will always be a number or `null`/`undefined`

Reply:
Should we change this to `!== undefined/null`?  
Then the (incorrect) string value would be passed into `FormattedNumber`. Maybe that's a less confusing error than showing "-" when the value is a string.